src/Elm/Syntax/Expression.elm

```diff
@@ -143,7 +143,8 @@ type alias Lambda =
 -}
 type alias CaseBlock =
     { expression : Node Expression
-    , cases : Cases
+    , firstCase : Case
+    , restOfCases : List Case
```

Comment:
Initially I had `cases : (Case, List Case)` but without requiring https://package.elm-lang.org/packages/turboMaCk/non-empty-list-alias/latest/ it felt awkward to use so I split it into two record fields instead.